runner/app/pipelines/audio_to_text.py

```diff
@@ -77,11 +76,19 @@ def __call__(self, audio: UploadFile, **kwargs) -> List[File]:
             converted_bytes = audio_converter.convert(audio, "mp3")
             audio_converter.write_bytes_to_file(converted_bytes, audio)
 
+        if kwargs["return_timestamps"] != "false":
```

Comment:
@eliteprox can you investigate in the [transformers library](https://github.com/huggingface/transformers) what options they have. Do they just have true and false? If so we can also just use those options.